### ISSUE TITLE
dont call gst_deinit twice

### ISSUE THREAD

**Issue by landryb** [baresip/baresip#1928]
if both gst and gst_video modules are loaded via config, baresip will crash at exit:
```
(baresip:16962): GStreamer-ERROR **: 15:18:14.040: GStreamer should not be deinitialized a second time.
```
i guess because `gst_deinit()` is called in both modules.

**Comment by alfredh** [comment]
I have tried this on MacOS and Linux, and no warnings is printed.
Which platform are you on ?

A better solution is to delete gst_video.so module, because I am not sure if anyone
is using it or maintaining it.

**Comment by landryb** [comment]
> I have tried this on MacOS and Linux, and no warnings is printed. Which platform are you on ?

OpenBSD.

> A better solution is to delete gst_video.so module, because I am not sure if anyone is using it or maintaining it.

Good call :)

**Comment by alfredh** [comment]
fixed in b43dd2003acbfd6b9a5fe6cb32c36cfb8dbb0774
